app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapView.java

```diff
@@ -36,6 +38,14 @@
     private CesiumVectorLayersBridge mCesiumVectorLayersBridge;
     private CesiumMapBridge mCesiumMapBridge;
     private CesiumKMLBridge mCesiumKMLBridge;
+    private OnGGMapReadyListener mOnGGMapReadyListener;
+
+    /**
+     * A synchronized data holder is used to allow multi-threaded scenarios
+     * that occur while JS thread is updating data, that's simultaneously used
+     * by another thread
+     */
+    private SynchronizedDataHolder<Boolean> mIsGGMapReadySynchronized;
```

Comment:
Both calback and a simple get method are available.
I thought I would need the get method performance-wise but it turned out I didn't.
This code can be deleted since It's not used, but it's quite common to have both an "event"/"listener"/callback to state changes AND a direct get method for that state.